Aspire Budgeting/SceneDelegate.swift

```diff
@@ -9,12 +9,14 @@ import UIKit
 
 final class SceneDelegate: UIResponder, UIWindowSceneDelegate {
   var window: UIWindow?
-  private let objectFactory = ObjectFactory()
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/d909f41c73c54120b2fbb4e96bb79560130cb154/Aspire Budgeting/SceneDelegate.swift#L12"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>SceneDelegate.swift:12</code></td>
    </tr>
  </tbody>
</table>